cdc/processor/pipeline/sorter.go

```diff
@@ -102,6 +114,45 @@ func (n *sorterNode) Init(ctx pipeline.NodeContext) error {
 				if msg == nil {
 					continue
 				}
+				if msg.RawKV != nil && msg.RawKV.OpType != model.OpTypeResolved {
+					size := uint64(msg.RawKV.ApproximateSize() * 2)
+					commitTs := msg.CRTs
+
+					if time.Since(lastSendResolvedTsTime) > time.Millisecond*200 {
```

Comment:
Could you please add some comments about how these three variables (`lastCRTs`, `lastSentResolvedTs`, `commitTs`) work together.

Reply:
Ok